src/autopas/containers/linkedCells/traversals/C04SoACellHandler.h

```diff
@@ -139,10 +138,10 @@ class C04SoACellHandler {
    * @param bufferSlice Index of slice in combinationSlice (source)
    * @param cellSlice Index of slice in _baseOffsets (destination)
    */
-  void writeBufferIntoCell(std::vector<ParticleCell> &cells, const unsigned long baseIndex,
+  void writeBufferIntoCell(std::vector<ParticleCell> &cells, unsigned long baseIndex,
```

Comment:
why remove const?

Reply:
because it doesn't have any effect here.
(const for copy by value has only an effect in function implementation, not the declaration)